include/quicr/detail/base_track_handler.h

```diff
@@ -19,9 +19,8 @@ namespace quicr {
     enum class TrackMode : uint8_t
     {
         kDatagram,
-        kStreamPerObject,
-        kStreamPerGroup,
-        kStreamPerTrack
+        kStreamPerTrack,
+        kStreamPerSubGroup,
```

Comment:
No more group? I thought `kStreamPerTrack` was the one going the way of the dodo?

Reply:
yes .. stream per group and stream per object gets subsumed by subgroups